win32ss/user/ntuser/main.c

```diff
@@ -807,6 +807,9 @@ ExitThreadCallback(PETHREAD Thread)
             UserDereferenceObject(ref->obj);
 
             psle = PopEntryList(&ptiCurrent->ReferencesList);
+#if DBG
+            ptiCurrent->cRefObjectCo--;
```

Comment:
I wonder whether having an assertion following this, that the cRefObjectCo cannot be < 0 (or <= 0 ??) may be of interest.

Reply:
> following this

Note that `cRefObjectCo` is added as a `UINT`, so such a check would have to be done before decrementing.

Reply:
that will bomb out on the ` ASSERT(ptiCurrent->cRefObjectCo == 0);` below anyway, so what is the point of adding more asserts that check the same?